spec/services/encrypted_document_storage/document_writer_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+RSpec.describe EncryptedDocumentStorage::DocumentWriter do
+  describe '#encrypt_and_write_document' do
+    it 'encrypts the document and writes it to storage' do
+      front_image = 'hello, i am the front image'
+      back_image = 'hello, i am the back image'
+
+      result = EncryptedDocumentStorage::DocumentWriter.new.encrypt_and_write_document(
+        front_image: front_image,
+        back_image: back_image,
+      )
```

Comment:
this reminds me, should we add some sort of `content_type` metadata for front + back image? so that consumers of the image data don't try to parse a PNG as a JPEG or something, etcetc

Reply:
What do you mean by adding a content type? Like adding something to the encrypted payload?

Reply:
no like another metadata value we log, so like
- front_image_uuid
- front_image_content_type
- front_image_key
- back_image_uuid
- back_image_content_type
- back_image_key # happens to be the same as front image key for now

Reply:
oh, you mean like something that gets written to the attempts API? We can do that, but attempts API is out of scope for this particular change.

Reply:
Yup! I was just thinking it would be additional keys on the `WriteDocumentResult`

Reply:
Let me see what I can do. It may actually be easier to add the content types where the attempts API `#track_event` call happens rather than here. We'd have to pass them all the way down to this point from the caller which will shortly call `track_event`.

Reply:
Yup! Could be a good candidate for a follow-up PR too, just figured I'd flag it here while I thought of it

Reply:
I added an AC to the jira ticket for adding the values to the attempts API

Reply:
@zachmargolis - What's the benefit here? 

> so that consumers of the image data don't try to parse a PNG as a JPEG or something, etcetc

We aren't streaming the image to anyone; it's stored in an S3 bucket, and will need to be manually accessed.

Reply:
Another alternative to preserving the content type would be to keep the extension from the original file (or embed that somehow)

- Right now, the filenames are "plain" UUIDs like `s3://bucket/eb2921d1-4cde-423f-bf77-ca298c29ba7a`
- I'm trying to imagine 3 months down the line, if we need to investigate or inspect one of these, somebody is going to need to download that `eb2921d1-4cde-423f-bf77-ca298c29ba7a` file, and after decrypting, they'll likely need to give it an extension to be able to view it, so preserving the content type helps us keep the useful data around to help that future workflow